tests/bears/BearTest.py

```diff
@@ -489,6 +498,18 @@ def test_debug_run_with_no_return(self, runcall):
         kwargs = {}
         self.assertIsNone(my_bear.run_bear_from_section(args, kwargs))
 
+    def test_do_settings(self):
+        section = Section('name', None)
+        section.append(Setting('x', '85'))
+        section.append(Setting('y', 'kbc3'))
+        section.append(Setting('z', '75'))
+        my_bear = TestOneBear(section, self.queue)
+        dbg = Debugger(bear=my_bear)
+        arg = ()
+        self.assertEqual(dbg.do_settings(arg), 1)
```

Comment:
without checking that the settings are actually printed this test is not very representative^^

Reply:
@Makman2  I think thats why we created a ideal test `DebugBearsTest` ?, I will try with mock and make test more effective